Makefile

```diff
@@ -24,4 +24,4 @@ infra-destroy:
 	terraform -chdir=infra destroy
 
 provision:
-	ansible-playbook ./provision/main.yml
+	ANSIBLE_CONFIG="./provision/ansible.cfg" ansible-playbook ./provision/docker.yml
```

Comment:
Not sure about the file name. Eventually, we do not use docker. How about keep the name `main`?

Reply:
yeah. makes sense